lib/sedna/algorithms/unseen_task_processing/unseen_task_processing.py

```diff
@@ -0,0 +1,157 @@
+import json
+
+from sedna.backend import set_backend
+from sedna.common.file_ops import FileOps
+from sedna.common.constant import KBResourceConstant
+from sedna.common.class_factory import ClassFactory, ClassType
+
+__all__ = ('UnseenTaskProcessing', )
+
+
+class UnseenTaskProcessing:
+    '''
+    Process unseen tasks given task update strategies
+
+    Parameters:
+    ----------
+    estimator: Instance
+        An instance with the high-level API that greatly simplifies
+        machine learning programming. Estimators encapsulate training,
+        evaluation, prediction, and exporting for your model.
+    cloud_knowledge_management: Instance of class CloudKnowledgeManagement
+    unseen_task_allocation: Dict
+        Mining tasks of unseen inference sample.
+    '''
+
+    def __init__(self, estimator, config,
+                 cloud_knowledge_management,
+                 edge_knowledge_management,
+                 unseen_task_allocation,
+                 **kwargs):
+        self.estimator = set_backend(estimator=estimator, config=config)
+        self.cloud_knowledge_management = cloud_knowledge_management
+        self.edge_knowledge_management = edge_knowledge_management
+
```

Comment:
put parameters（`cloud_knowledge_management` and `edge_knowledge_management`）to other funs instead of initial func.